src/service/action-impl.js

```diff
@@ -200,6 +204,15 @@ export class ActionService {
         this.addChangeDetails_(event);
         this.trigger(dev().assertElement(event.target), name, event);
       });
+    } else if (name == 'input-debounced') {
+      const inputDebounced = debounce(this.ampdoc.win, (target, event) => {
+        this.trigger(
+            dev().assertElement(target), name, /** @type {?Event} */ (event));
+      }, DEFAULT_DEBOUNCE_WAIT);
+
+      this.root_.addEventListener('input', event => {
+        inputDebounced(event.target, event);
```

Comment:
Not yet.

```js
function cloneEvent(event) {
  const clone = map();
  for (const prop in event) {
    const value = event[prop];
    // Avoid functions like stopPropagation, since they can't do anything in an async context.
    if (typeof value !== 'function') {
      clone[prop] = event[prop];
    }
  }
}